cves/kernel/CVE-2021-3744.yml

```diff
@@ -225,7 +229,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: drivers
```

Comment:
could `crypto` also be an additional subsystem?

Reply:
You're right